src/windows/main_window.py

```diff
@@ -1021,6 +1021,12 @@ def actionSaveFrame_trigger(self, event):
             self.statusBar.showMessage(_("Save Frame cancelled..."), 5000)
             return
 
+        # Set scale mode to higher quality (to skip chroma optimizations of preview)
+        openshot.Settings.Instance().HIGH_QUALITY_SCALING = True
```

Comment:
Other than fixing the traceback, the only thing I'd suggest is storing the old value here, and restoring it after — so that, if it's _not_ `False` already, you don't accidentally **force** it false.
```suggestion
        prev_scaling = openshot.Settings.Instance().HIGH_QUALITY_SCALING
        openshot.Settings.Instance().HIGH_QUALITY_SCALING = True
```

Reply:
Changes not required. It is set during init of the main window and switched back and force only in Export.

Reply:
For now! But can you guarantee that code won't be added in the future which sets it to `True` somewhere, for some reason? (Or maybe it's just _always_ set to `True`?)

Why should whoever makes that change then have to go and find **_this_** code, so they can fix it to stop forcibly resetting the value to `False`? It can easily be made safe for any existing value of the setting, and then we _do_ have a guarantee that there will be no such problem in the future.

My favorite kind of future problems are the ones that get solved before they even happened.

Reply:
If I would secure any possibilities - the only thing this application will do is checking, checking, and checking. Without any useful result in the end. OpenShot is by design has low quality until saving. I'm just using this rule.